### DISCUSSION TITLE
Support request:

### DISCUSSION THREAD

**Discussion by kwezi-njikelana-rentoza** [aws/codewhisperer-command-line-discussions#98] [Support ticket]
### Checks

- [X] I ran `cw doctor` in the affected command line session and it didn't resolve my issue
- [X] I ran `cw restart` and it didn't resolve my issue
- [X] I searched [the discussions in this repo](https://github.com/aws/codewhisperer-command-line-discussions/discussions?q=) and there are no duplicates of my issue

### Expected behaviour

Fir cw to wok when i run my commands on my terminal

### Actual behaviour

It's still not working

### Steps to reproduce

No

**Comment by grant0417** [comment]
Can you give some more details? Going to close unless you can share more of what is happening.
